backend/crates/eiffelvis_gen/src/event.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct Data {}
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct Meta {
+    pub id: Uuid,
+    #[serde(rename = "type")]
+    pub event_type: String,
+    pub version: String,
+    pub time: u64, // Not high priority?
+                   // source: Source
+                   // security: Security
+}
+
+#[derive(Default, Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+pub struct Link {
+    #[serde(rename = "type")]
+    pub link_type: String,
+    pub target: Uuid,
+}
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct Event {
```

Comment:
This should probably be an internal type as it should not really be used by consumers